web/src/settings_org.ts

```diff
@@ -271,90 +271,8 @@ function set_msg_edit_limit_dropdown(): void {
     settings_components.set_time_limit_setting("realm_message_content_edit_limit_seconds");
 }
 
-function message_move_limit_setting_enabled(
-    related_setting_name:
-        | "realm_can_move_messages_between_topics_group"
-        | "realm_can_move_messages_between_channels_group",
-): boolean {
-    const user_group_id = settings_components.get_dropdown_list_widget_setting_value(
-        $(`#id_${related_setting_name}`),
-    );
-    assert(typeof user_group_id === "number");
-    const user_group_name = user_groups.get_user_group_from_id(user_group_id).name;
-    if (
-        user_group_name === "role:administrators" ||
-        user_group_name === "role:moderators" ||
-        user_group_name === "role:nobody"
-    ) {
-        return false;
-    }
-
-    return true;
-}
```

Comment:
This is reasonable. I think in the future, if desired, we can add a "group who have a time limit for editing messages" type of additional setting.